src/components/MenuButtons.js

```diff
@@ -1,44 +1,123 @@
-import Link from 'next/link';
+import Link from '@material-ui/core/Link';
```

Comment:
Out of curiosity, what are the differences between the nextjs `Link` component and material UI's?

Reply:
Also if we're going to go the route of using both components, let's abstract it to a new custom `Link` component in utils. Also since we only have components in `utils` directory, let's move it into the `components` directory.

Reply:
I think I accidentally auto imported one of those links haha. We should only need the one.

Reply:
I switched back to next/link and fixed the styling on the current Navbar. Instead of moving it to the components directory, would it make more sense to move Space and Media query to utils and keep Wrapper and MenuButtons in components?

It seems to describe them better that way in my mind and either folder may grow as we go.

Reply:
Yeah makes sense to move those into utils.